addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -258,7 +258,7 @@ class GVAR(actions) {
         displayNameProgress = CSTRING(DiggingGrave);
         icon = QPATHTOEF(medical_gui,ui\cross_grave.paa);
         treatmentTime = QGVAR(treatmentTimeGrave);
-        condition = QFUNC(canDigGrave);
+        condition = QUOTE(!([_this#1] call EFUNC(common,isAwake)) && {_this call FUNC(canDigGrave)});
```

Comment:
```suggestion
        condition = QFUNC(canDigGrave);
```

Reply:
I think less code in configs is better

Reply:
Sure we can do it this way if you'd prefer